packages/frontend/src/views/HomeView.vue

```diff
@@ -7,34 +7,24 @@
         due to a lack of resources. Use the Dependency Health Monitor to view the financial state of your dependencies
         and support your dependency!
       </h4>
-      <Button @click="redirectToExecuteReportPage" label="Execute the report!" icon="pi pi-heart" />
+      <a class="no-underline" href="/run-report/select">
+        <Button label="Execute the report!" icon="pi pi-heart" />
+      </a>
     </div>
     <div id="help-improve" class="col-6">
       <IconHelpImproveVue />
       <div class="flex justify-content-center">
-        <Button
-          @click="openStorySetPage"
-          label="People illustrations by Storyset"
-          class="p-button-text"
-          icon="pi pi-external-link"
-        />
+        <a class="no-underline" href="https://storyset.com/people" target="_blank" rel="noopener noreferer">
```

Comment:
`Button` component will render a `button` HTML element, right? If so, this isn't allowed. The best thing we can do here, semantically, is copy the HTML output from the primevue `Button` component and replace the HTML `button` element with an `a` element

Reply:
See **Permitted Content** in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#properties

Reply:
Resolved in 7c4733217c8f0f384c72d18f886503816ff1a733